tools/benchmark/src/ResultTypes.ts

```diff
@@ -64,7 +64,7 @@ export interface BenchmarkData {
 	/**
 	 * Custom data formatters.
 	 */
-	customDataFormatters: Record<string, (value: unknown) => string>;
+	customDataFormatters: Record<string, string>;
```

Comment:
Is this intended to be reassigned after being set? If not
```suggestion
	readonly customDataFormatters: Record<string, string>;
```

Reply:
`customDataFormatters` removed so no longer applies 😄 .